lib/ClangImporter/ClangImporter.cpp

```diff
@@ -683,16 +683,9 @@ importer::getNormalInvocationArguments(
     invocationArgStrs.push_back("-fmodules-validate-system-headers");
   }
 
-  if (importerOpts.DetailedPreprocessingRecord) {
-    invocationArgStrs.insert(invocationArgStrs.end(), {
-      "-Xclang", "-detailed-preprocessing-record",
-      "-Xclang", "-fmodule-format=raw",
-    });
```

Comment:
I don’t know what we used this for but why do we no longer need it?

Reply:
I would expect failures here, or maybe in the eventual integration tests if we have one for this case. As far as I can tell, sourcekitd doesn't link with the LLVM backend. So we shouldn't be able to generate debug info. Unless it's linked through one of the dependencies and I'm just missing that.

In any case, we don't have any real reason *to* output debug info, so my preference would be to remove `-detailed-preprocessing-record` but leave in `-fmodule-format=raw`. And probably to rename the importer option (it's only real read is here, but there's a bunch of places that need updating).